inmap/cmd/config.go

```diff
@@ -119,14 +121,24 @@ func ReadConfigFile(filename string) (config *ConfigData, err error) {
 	}
 
 	for k, v := range config.OutputVariables {
+		v = strings.Replace(v, "\r\n", " ", -1)
```

Comment:
If the first function deletes "\r" and the second function deletes "\n", then running both functions will also delete "\r\n", right?